elixir/apps/domain/lib/domain/auth/adapters/okta/api_client.ex

```diff
@@ -100,13 +100,19 @@ defmodule Domain.Auth.Adapters.Okta.APIClient do
     end
   end
 
+  if Mix.env() == :test do
```

Comment:
@bmanifold this trick allows to override the behaviour per environment. We should not put static sleeps anywhere - instead they should be either in a function that is overridden (like here, for temporary code that will be removed) or added to application environment (that we can override in `test.exs`).

Reply:
👍 makes sense.  When I added the sleep I was only thinking about making sure we weren't causing any rate limits to be triggered, but this is definitely a good thing for me to keep in mind.

Reply:
Out of curiosity, was this causing issues in the tests?  Or just slowing the tests down?

Reply:
Yeah, some cases were taking 10+ seconds by themselves while all suite before took about the same time.

Reply:
Ah, yeah, that's not good.  Sorry about that.  I think I must have let CI run the tests for me when I pushed that change, so I didn't feel the immediate pain of the tests taking longer.

Reply:
No worries, I just brought it up for the future